grafana-plugin/e2e-tests/utils/userSettings.ts

```diff
@@ -69,18 +66,6 @@ export const configureUserNotificationSettings = async (page: Page, notifyBy: No
   // open the user settings modal
   await openUserSettingsModal(page);
 
-  /**
-   * see if we already have a default notification setting
-   * if we don't click the Add Notification Step button and add one
-   * otherwise update the existing one
-   */
-  const defaultNotificationsAddNotificationStepButton = page.locator(
-    'div[class*="PersonalNotificationSettings"] >> nth=0 text=Add Notification Step'
-  );
-  if (await defaultNotificationsAddNotificationStepButton.isVisible()) {
-    await defaultNotificationsAddNotificationStepButton.click();
-  }
-
```

Comment:
we can get rid of this because now we always have email set as the first default (and important) personal notification step (and we don't allow personal notification policies to be empty; ie you can't delete the last step)